index.js

```diff
@@ -514,7 +514,7 @@ function launchApp(argo, startExternalAdapterServer) {
         if (openfinWinOpts && !isRunning) {
             //making sure that if a window is pressent we set the window name === to the uuid as per 5.0
             openfinWinOpts.name = uuid;
```

Comment:
👍 @nicholasdgoodman have a look at the update

[updated tests look good](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5953b00b0c101702173658d6) fails pass in isolation